README.md

```diff
@@ -37,7 +37,7 @@ Internet at large!
   | Field | Value
   | ----: | :----
   | Type  | A
-  | TTL   | 5 minutes
+  | TTL   | 600 seconds
```

Comment:
Let's leave this as the original value since I don't see a good reason to change to 10 minutes instead of 5.